keg_auth/model/__init__.py

```diff
@@ -490,8 +490,8 @@ class BundleMixin(object):
     @might_commit
     @might_flush
     @classmethod
-    def edit(cls, oid=None, **kwargs):
-        obj = super(BundleMixin, cls).edit(oid=oid, _commit=False, **kwargs)
+    def edit(cls, id=None, **kwargs):
```

Comment:
nit: the reason I don't use "id" as a variable is because I don't want to shadow the id built-in function.